docs/env-var-reference.md

```
@@ -98,23 +98,57 @@ Override the environment tag sent with telemetry data. By default, CumulusCI
+in the org rather than bugs in CumulusCI itself.
+
+| Value | Behaviour |
+|---|---|
+| `always`, `1`, `true`, `yes` | Always send without prompting |
+| `never`, `0`, `false`, `no` | Never send, never ask |
+| *(not set)* | Prompt in interactive sessions; auto-send in CI |
+
+In interactive sessions, if this variable is not set, CumulusCI will ask
+after displaying the error:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)